astroplan/constraints.py

```diff
@@ -997,24 +997,37 @@ def observability_table(constraints, observer, targets, times=None,
     return tab
 
 
-def _rescale_minmax(vals, min_val, max_val):
-    """ Rescale altitude into an observability score."""
-    rescaled = (vals - min_val) / (max_val - min_val)
-    below = rescaled < 0
-    above = rescaled > 1
-    rescaled[below] = 0
-    rescaled[above] = 1
-
-    return rescaled
+def _rescale_minmax(vals, worst_val, best_val, better_than=1, worse_than=0):
+    """
+    rescales the input ``vals`` between 0 and 1
 
+    Parameters
+    ----------
+    vals : array of values
+        the values that need to be rescaled to be between 0 and 1
+    worst_val : value
+        worst acceptable value (rescales to 0)
+    best_val : value
+        best value cared about (rescales to 1)
+    better_than : 0 or 1
+        what is returned for ``vals`` beyond the ``best_val``
+    worse_than : 0 or 1
+        what is returned for ``vals`` beyond the ``worst_val``
 
```

Comment:
Would you include an example or two demonstrating how it works? Here's an [example](https://github.com/astropy/astroplan/blob/master/astroplan/observer.py#L317-L333) to refer to for the format. For example, you could use `vals = altitude = [0, 10, 20]` and `vals = airmass = [3, 2, 1]` and show how the outputs match what you'd expect.